e2e_playwright/st_heading_test.py

```diff
@@ -17,7 +17,10 @@
 import pytest
 from playwright.sync_api import Locator, Page, expect
 
-from e2e_playwright.conftest import ImageCompareFunction, wait_for_app_loaded
+from e2e_playwright.conftest import (
```

Comment:
nit/question:
do we not have auto-formatting running properly for our e2e tests?

Reply:
We have, but if there is a comma at the last position, it will keep the multi-line format. I probably added and removed an import and forgot to clean up the comma again.

Reply:
Updated it to fix the formatting

Reply:
Thanks for clarifying!